cli/server.go

```diff
@@ -132,7 +133,37 @@ func server() *cobra.Command {
 				}
 			}
 
-			printLogo(cmd, spooky)
+			config := createConfig(cmd)
+			// For in-memory, disable embedded PostgreSQL!
+			// This is primarily for testing and development.
+			if inMemoryDatabase {
+				postgresBuiltin = false
+				postgresURL = ""
+			}
+			// Only use built-in if PostgreSQL URL isn't specified!
+			if postgresURL == "" && postgresBuiltin {
+				var closeFunc func() error
+				postgresURL, closeFunc, err = startBuiltinPostgres(cmd.Context(), config, logger)
+				if err != nil {
+					return err
+				}
+				cmd.Printf("Using built-in PostgreSQL (%s)\n", config.PostgresPath())
+				defer func() {
+					// Gracefully shut PostgreSQL down!
+					_ = closeFunc()
+				}()
+			}
+			// If the built-in PostgreSQL isn't used and an external URL isn't specified,
+			// recommend usage of the built-in!
+			if !inMemoryDatabase && postgresURL == "" {
+				cmd.PrintErrln(cliui.Styles.Error.Render("PostgreSQL >=13 is required for running Coder.") + "\n")
```

Comment:
```suggestion
				cmd.PrintErrln(cliui.Styles.Error.Render("PostgreSQL >=13 is required to run Coder.") + "\n")
```

Minor grammar suggestion, otherwise it may sound like PostgreSQL runs Coder 😄.